README.md

```diff
@@ -284,6 +288,9 @@ non-italicized when typing the letters without typing a backslash first, like
 3.17 of the Short Math Guide][3.17]) plus some missing trig operators like
 `sech`, `arcsec`, `arsinh`, etc.
 
+You can also specify a speech-friendly representation of the operator name by supplying the operator name, a `|` and its speech alternative (separate multiple words with a `-`). For example, 'sin|sine cos|cosine tan|tangent sinh|hyperbolic-sine'.
```

Comment:
What about people that don't specify any autoOperators (so that users need to type a backslash before sin to get the operator form). It seems like MathQuill might want to have a default concept of how the operators that it knows about should be spoken.